perses/tests/test_topology_proposal.py

```diff
@@ -905,9 +905,60 @@ def test_simple_heterocycle_mapping(iupac_pairs = [('benzene', 'pyridine')]):
 
         assert num_hetero_maps > 0, f"there are no differences in atomic number mappings in {iupac_pair}"
 
+def test_protein_counterion_topology_fix():
```

Comment:
I didn't benchmark this test's run time but if it is "short" then can we add test cases where the charge +1, and zero?

Reply:
Sure, feel free to add those.